packages/analyzer/src/html/html-document.ts

```diff
@@ -204,28 +206,38 @@ export class ParsedHtmlDocument extends ParsedDocument<ASTNode, HtmlVisitor> {
       const docContainingNode = doc.astNode.node;
       const docContainingLocation = docContainingNode.__location;
       let expectedIndentation;
-      if (docContainingLocation &&
-          !isElementLocationInfo(docContainingLocation)) {
-        expectedIndentation = docContainingLocation.col;
-
-        const parentLocation = docContainingNode.parentNode &&
-            docContainingNode.parentNode.__location;
-        if (parentLocation && !isElementLocationInfo(parentLocation)) {
-          expectedIndentation -= parentLocation.col;
+      if (prettyPrint) {
+        if (docContainingLocation &&
+            !isElementLocationInfo(docContainingLocation)) {
+          expectedIndentation = docContainingLocation.col;
+
+          const parentLocation = docContainingNode.parentNode &&
+              docContainingNode.parentNode.__location;
+          if (parentLocation && !isElementLocationInfo(parentLocation)) {
+            expectedIndentation -= parentLocation.col;
+          }
+        }
+        if (expectedIndentation === undefined) {
+          expectedIndentation = 2;
         }
-      }
-      if (expectedIndentation === undefined) {
-        expectedIndentation = 2;
       }
 
       // update the cloned copy of docContainingNode with the
       // inlined document stringification.
       const clonedDocContainingNode =
           this._findClonedContainingNode(astClone, docContainingNode);
-      dom5.setTextContent(clonedDocContainingNode!, '\n' + doc.stringify({
-        indent: expectedIndentation,
-        inlineDocuments: otherDocuments
-      }) + '  '.repeat(expectedIndentation - 1));
+
+      const inlineStringifyOptions = Object.assign(
+          {},
```

Comment:
Use `{...options, {indent: /* etc */}` syntax here?

Reply:
Hmm, it _appeared_ that I couldn't with the tsconfig, but after rebooting VSCode it works now.